openmicroscopy/openmicroscopy
# Spw download enabled develop

These are the additional commits to #3351 backported to develop.

To test:

 - Download of SPW data should be disabled by default, both when a single well is selected and when multiple wells are selected (batch annotate panel). Download of non-SPW images should still be working.
 - After configuration and restart of web: ``` $ bin/omero config set omero.web.plate_download.enabled True ```
Now, download should be enabled for single or multiple selected wells, BUT OME-TIFF export should still be disabled for all SPW data.
 - Finally, check that OME-TIFF is still enabled for non-SPW data.

NB: Need to switch the error for archived_files/download from 500 to 404 when the specified images have no archived files.

--rebased-from #3351.
